source/content/drush.md

```diff
@@ -20,48 +20,80 @@ You can run all of the commands below from Terminus instead of using Drush alias
 For details on managing remote and local Drush versions, see [Managing Drush Versions on Pantheon](/drush-versions).
 
 ## Download Drush Aliases Locally
-Downloading the Pantheon aliases to your local Drush aliases file allows you to run Drush calls against your Pantheon site environments.
+Downloading the Pantheon aliases to your local Drush aliases file allows you to run Drush calls against your Pantheon site environments. Use [Terminus](/terminus/) to download your Drush aliases.
 
-There are two ways to obtain the aliases, either with Terminus or through the Dashboard.
-
-### Download with Terminus, the Pantheon CLI
-Authenticate Terminus with [machine tokens](/machine-tokens/) or your Pantheon Dashboard credentials, then update your local aliases file in a single step:
+Authenticate Terminus with [machine tokens](/docs/machine-tokens/) or your Pantheon Dashboard credentials, then update your local aliases file in a single step:
 
+```bash
+terminus aliases
 ```
-$ terminus aliases
+
+This command will write both Drush 8 and Drush 9 aliases into the directory `$HOME/.drush` for your sites that you are a direct member of. To create aliases for all sites that you can use, including sites that you have access to via team membership, run:
+
+```bash
+terminus aliases --all
 ```
 
-### Download Using the Dashboard
+If you add a site to your account, you will have to download a new copy of your Drush aliases. You do not need to update your Drush aliases when you add new mulitdev environments to your sites.
```

Comment:
Is this true for Drush 8 too?

Reply:
Yes, if using the new version of Terminus and Drush 8.3.0+